packages/components/action-button/src/utils/constants.js

```diff
@@ -0,0 +1,11 @@
+export const types = {
+	gray: 'gray',
+	blue: 'blue',
+	inverted: 'inverted'
+};
+
+export const actionTypeToIconType = {
+	gray: 'action-gray',
+	blue: 'action-blue',
+	inverted: 'action-inverted'
```

Comment:
I suggest to change the keys to UPPERCASE for consistency with the `ts-button` component types.

Reply:
Good idea :+1: 
I realized it's not constants, it's a _dictionary_ which maps type of _ts-action-button_ with the type of _ts-icon_, so keys here must be strings. 

Anyway, this code is removed, because a new types of _action-button_ and _ts-icon_ maps perfectly without any conversion.